.github/workflows/ci.yml

```diff
@@ -84,3 +84,15 @@ jobs:
       run: cargo install cargo-audit
     - name: Run audit command
       run: cargo audit ${{ env.CARGO_AUDIT_BASE_FLAGS }}
+  credentials-scan:
+    name: Credentials scan
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: Secret Scanning
+      uses: trufflesecurity/trufflehog@main
+      with:
+        extra_args: --only-verified
```

Comment:
The `--only-verified` argument means this workflow only shows valid (verified) credentials.

https://github.com/trufflesecurity/trufflehog?tab=readme-ov-file#what-is-credential-verification
> What is credential verification?
>
> For every potential credential that is detected, we've painstakingly implemented programmatic verification against the API that we think it belongs to. Verification eliminates false positives. For example, the [AWS credential detector](https://github.com/trufflesecurity/trufflehog/blob/main/pkg/detectors/aws/aws.go) performs a GetCallerIdentity API call against the AWS API to verify if an AWS credential is active.

Reply:
We should not call any API on the behalf of the user even if it is used only for verification purpose. It introduces additional concern, for example, it may be used to attack the user who accidentally expose their credential.